page/events/introduction-to-custom-events.md

```diff
@@ -68,11 +73,11 @@ $( ".lightbulb" ).on( "changeState", function( e ) {
 });
 
 $( ".switch, .clapper" ).click(function() {
-	$( this ).parent().find( ".lightbulb" ).trigger( "changeState" );
+	$( this ).parent().find( ".lightbulb" ).trigger( "changeTheLightState" );
```

Comment:
We already have a better name now, committed in https://github.com/jquery/learn.jquery.com/commit/f9d0ab36e48de77aa9b5fc8f137bb2d86280d39d.